packages/ui/tailwind-utils-config/components/pagination.ts

```diff
@@ -0,0 +1,50 @@
+export default {
+  '.cn-pagination': {
+    '&-root': {
+      containerType: 'inline-size',
+      // TODO: Design system: How to handle magic numbers?
+      '@apply mx-auto w-full max-w-[700px]': '',
+
+      // Container query to hide pagination pages on smaller spaces
+      '@container (max-width: 400px)': {
```

Comment:
container query that hides pagination pages when the container shrinks below `400px`.